extensions/amp-story/0.1/amp-story.js

```diff
@@ -279,6 +279,10 @@ export class AmpStory extends AMP.BaseElement {
         () => this.variableService_);
 
     this.buildLandscapeOrientationOverlay_();
+
+    // Remove the cursor: pointer; style set by the runtime, to avoid wrong
+    // touch feedback on mobile, like a flashing overlay on page transitions.
+    resetStyles(this.win.document.documentElement, ['cursor']);
```

Comment:
I think it should be sufficient to do this in the "standalone" case (where amp-story is the only tag in the document), where we add the CSS class `.i-amphtml-story-standalone` to the document element anyway.  So, maybe in the CSS we can just add:

    .i-amphtml-story-standalone {
      cursor: default !important;
    }

Reply:
Done